### DISCUSSION TITLE
entities all go 'unavailable' immediately after viewing status of any device

### DISCUSSION THREAD

**Discussion by JamesMarcum** [mKeRix/room-assistant#748] [Q&A]
So I have installed 3 pi 0 Ws around the house, and am running room-assistant add-on on my HA instance as well (Pi 4b).  I have a cluster set up with all of them using BT Classic and HA integrations.  I haven't added any of the entities to the dashboard yet, since I am still working out the kinks, but whenever I go to MQTT and view any of the RA instances or tracked devices, they show initially and the almost immediately all are marked as 'unavailable' for about a minute.  I am still a bit new to HA, so the only logs I know to look in are the system ones in the UI (Supervisor, Server Controls).  Neither of these shows anything wrong that I can tell.  Any pointers?

One of my local.yml files -

global:
  instanceName: living-room
  integrations:
    - homeAssistant
    - bluetoothClassic
cluster:
  autoDiscovery: false
  weight: 50
  port: 6425
  peerAddresses:
    - 10.0.0.101:6425
    - 10.0.0.102:6425
    - 10.0.0.103:6425
    - 10.0.0.104:6425
homeAssistant:
  mqttUrl: mqtt://10.0.0.101:1883
  mqttOptions:
    username: fakeuser
    password: fakepwd123
  discoveryPrefix: homeassistant
bluetoothClassic:
  interval: 15
  addresses:
    - XX:XX:XX:XX:XX:XX # Obviously changed for this example, but there are 6 total addresses

(also note - my spacing is correct, the forum cropped out the leading spaces)

**Comment by B1n00** [comment]
I observed also the same behaviour device.tracker are being updated every 2 minutes with 'unvailable' status. While looking at MQTT Explorer it seems that this behaviour is caused by availability mode message that is being sent in the same topic than the state

States message:
`{"payload_available":"online","payload_not_available":"offline","name":"ble_presence Tracker","device":{"identifiers":"XX-XX-XXX-XXXX-XXX","name":"ble_presence","via_device":"room-assistant-distributed"},"unique_id":"room-assistant-ble-"XX-XX-XXX-XXXX-XXX-tracker","state_topic":"room-assistant/device_tracker/ble-"XX-XX-XXX-XXXX-XXX-tracker/state","json_attributes_topic":"room-assistant/device_tracker/ble-"XX-XX-XXX-XXXX-XXX-tracker/attributes","availability_topic":"room-assistant/device_tracker/ble-"XX-XX-XXX-XXXX-XXX-tracker/status","payload_home":"home","payload_not_home":"not_home","source_type":"bluetooth_le"}`

Availability message:
`{"availability_mode":"all","name":"ble_presence Tracker","device":{"identifiers":"XX-XX-XXX-XXXX-XXX","name":"ble_presence","via_device":"room-assistant-distributed"},"unique_id":"room-assistant-ble-XX-XX-XXX-XXXX-XXX-tracker","state_topic":"room-assistant/device_tracker/ble-XX-XX-XXX-XXXX-XXX-tracker/state","json_attributes_topic":"room-assistant/device_tracker/ble-XX-XX-XXX-XXXX-XXX-tracker/attributes","payload_home":"home","payload_not_home":"not_home","source_type":"bluetooth_le","availability":[{"topic":"room-assistant/device_tracker/ble-XX-XX-XXX-XXXX-XXX-tracker/status","payload_available":"online","payload_not_available":"offline"},{"topic":"room-assistant/status/bureau","payload_available":"online","payload_not_available":"offline"}]}`

Is there some way to get rid of the availability message?

**Reply by B1n00** [reply]
I checked across my cluster (2 rpi 4 And 4 rpi zero) all updated to 2.18.4 version and I observed that according the model the message sent in the mqtt topic is not the same. I have the new template with the availability in the rpi4 and the pi,zero havé the old one.
Is there something specific to do on the pi zero?
